tests/js/server/dump/dump-smartattribute-regression-cluster.js

```diff
@@ -0,0 +1,82 @@
+/*jshint globalstrict:false, strict:false, maxlen:4000 */
+/*global assertEqual, assertTrue, assertFalse */
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief tests for dump/reload
+///
+/// @file
+///
+/// DISCLAIMER
+///
+/// @author Copyright 2020, ArangoDB Inc., Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Markus Pfeiffer
+/// @author Copyright 2020, ArangoDB Inc., Cologne, Germany
+////////////////////////////////////////////////////////////////////////////////
+
+'use strict';
+
+const db = require("@arangodb").db;
+var internal = require("internal");
+var jsunity = require("jsunity");
+
+const isEnterprise = internal.isEnterprise();
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief test suite
+////////////////////////////////////////////////////////////////////////////////
+
+function dumpTestSuite () {
+  'use strict';
+  var db = internal.db;
+
+  return {
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief set up
+////////////////////////////////////////////////////////////////////////////////
+
+    setUp : function () {
+    },
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief tear down
+////////////////////////////////////////////////////////////////////////////////
+
+    tearDown : function () {
+    },
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief test the empty collection
+////////////////////////////////////////////////////////////////////////////////
+    
+    testSmartGraphAttribute : function () {
```

Comment:
Can you move this test into dump-<engine>-cluster files instead?
This way we get away without a new TestSuite.